paper.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Trash AI: A Web GUI for Serverless Computer Vision Analysis of Images of Trash'
+tags:
+  - tensorflow.js
+  - IndexDB
+  - Plastic Pollution
+  - Trash
+  - Litter
+  - AI
+  - Image Classification
+  - Serverless
```

Comment:
Since we're discussing specific tech I'd suggest adding
* vue
* vuetify
* vite
* pinia

Reply:
Thanks @shollingsworth got these incorporated. https://github.com/code4sac/trash-ai/commit/3652a152ea3d3afb7e6df823f0791b1bb0e081a4